griffin/griffin.c

```diff
@@ -1466,6 +1466,7 @@ SSL
 ============================================================ */
 #if defined(HAVE_SSL)
 #if defined(HAVE_NETWORKING)
+#if defined(HAVE_BUILTINMBEDTLS)
```

Comment:
I think this effectively disables SSL for all griffin builds besides the ones you explicitly added `HAVE_BUILTINMBEDTLS` for in this PR (3DS and OSX?). Doing it this way means all the other project files now also need modification, like all the msvc's and other stuff.

Reply:
Also not that these build systems are already doing this for `-DHAVE_BUILTINMINIUPNPC`.

Reply:
Ok I thought it was enabled for at least MSVC but I guess not. Basically I think it just comes down to whether or not we want to require two ifdefs to get SSL to work correctly in that case or not. I'll leave it up to @twinaphex.

Reply:
I still think they would need it for this regardless of griffin?

https://github.com/libretro/RetroArch/pull/7745/files#diff-bb3428e6a736248b18d6df7696f75b19R28